libhb/decavsub.c

```diff
@@ -479,7 +479,7 @@ int decavsubWork( hb_avsub_context_t * ctx,
             // get translated to SSA
             //
             // When using the "real_time" option with CC608 subtitles,
-            // ffmpeg prepends an ASS rect that has only the preample
+            // ffmpeg prepends an ASS rect that has only the preamble
```

Comment:
I'm not entirely certain, it might be a typo for pre-sample. That's the problem w/ typos